src/lib/index.d.ts

```diff
@@ -25,6 +26,16 @@ export type WebsocketAdapterConfig = {
   }
 }
 
+export type HttpAdapterConfig = {
+  server: {
+    httpServer?: any,
+    adapter?: HttpServerType | typeof GleeAdapter
+    port?: number
+  },
+  client?: {
+    endpoint: any
+  }
+}
```

Comment:
I don't think this and the documentation match, can you check it once, in `docs`you have `query` and body by here you have `endpoint`.